### DISCUSSION TITLE
Can add an option for "Enable B-frames" ?

### DISCUSSION THREAD

**Discussion by jexwang** [HaishinKit/HaishinKit.swift#1042] [Q&A] [answered]
Hello,
I want to let user set kVTCompressionPropertyKey_AllowFrameReordering by itself,
can you add an option to set it instead just reference profileLevel is "Baseline"?

[Here](https://github.com/shogo4405/HaishinKit.swift/issues/958#issuecomment-1164332854) has a related responses.

Very thankful.

**Comment by shogo4405** [comment, marked as answer]
No. Baseline has no B-frame frame feature.
- https://en.wikipedia.org/wiki/Advanced_Video_Coding

**Reply by jexwang** [reply]
Sorry, I mean when profileLevel is "High",
When the user sets profileLevel to "High", maybe they want to disable the B-frame feature,
but they can't control kVTCompressionPropertyKey_AllowFrameReordering.

Thanks for your reply 🙂

**Reply by shogo4405** [reply]
Ok.

**Reply by shogo4405** [reply]
I accepted this PR [Feature: Expose key frame reordering as an option](https://github.com/shogo4405/HaishinKit.swift/pull/1050).
This PR contains option for "Enable B-frames".

Thank you @allan-o3h
